components/Avatar/Avatar.css

```diff
@@ -0,0 +1,154 @@
+@import url('https://fonts.googleapis.com/css2?family=Dongle&family=Open+Sans:ital@1&family=Source+Sans+Pro:ital,wght@0,700;1,400&display=swap');
+@import url("../../style.css");
+@import url("../../color-palette/color.css");
+
+*, body{
+    margin:0;
+    padding:0;
+}
+.tab-header{
+    top:20px;
+}
+.main-menu{
+    background-color: var(--white);
+    border-top-right-radius: 10px;
+    border-bottom-right-radius: 10px;
+    width: 15.625rem;
+    height: 100vh;
+    position: fixed;
+    top:10vh;
+    left:0;
+    white-space: nowrap;
+    color:var(--primary-color);
+    overflow-y: scroll;
+    border: 3px solid var(--primary-color);
+}
+
+.text-decoration{
+    text-decoration:none;
+    color: var(--primary-color);
+    padding: 20px 60px;
+    font-weight: 600;
+    font-size: 20px; 
+}
+
+
+.main-menu  .menu-item {
+    padding: 20px;
+    font-weight: 600;
+    font-size: 20px;   
+}
+.main-menu .menu-item:hover{
+    background-color: var(--primary-color);
+    color: var(--white);
+    cursor: pointer;
+}
+
+
+
+.text-decoration:hover{
+    text-decoration: none;
+    color: var(--white);
+}
+.primary-components-2{
+    margin-bottom: 20px;
+}
+::-webkit-scrollbar {
+    width: 4px;
+}
+
+::-webkit-scrollbar-track {
+    background: transparent;
+}
+  
+::-webkit-scrollbar-thumb {
+    background: var(--primary-color);
+    border-radius: 10px;
+}
+
+.component-library-components-container {
+    position: relative;
+    display: flex;
+    flex-flow: row wrap;
+    
+}
+
+.avatar-contents-square{
+    position: absolute;
+    margin-left: 16.875rem;
+    margin-top: 6.25rem;
+    margin-right: auto;
+}
+
+.avatar-contents-circle{
+    position: absolute;
+    margin-left: 16.875rem;
+    margin-top: 65.625rem;
+    margin-right: auto;
+}
+
+.avatar-sm-size {
+    width: 3rem;
+    height: 3rem;
+    font-size: 1rem;
+}
+
+.avatar-md-size {
+    width: 6.25rem;
+    height: 6.25rem;
+    font-size: 2rem;
+}
+
+.avatar-lg-size {
+    width: 10rem;
+    height: 10rem;
+    font-size: 3rem;
+}
+
+.avatar-space{
+    margin-left:25px;
+    margin-bottom: 30px;
+    margin-top: 15px;
+}
```

Comment:
You coud have used short-hand property for margin
like this =>   margin : 15px   0px   30px     25px;
                                   top   right   bottom   left